internal/resto/structs.go

```diff
@@ -0,0 +1,36 @@
+package resto
+
+// Details represent API response regarding job
+type Details struct {
```

Comment:
Heavy use of `interface{}` here. Isn't `Passed` a boolen or at the very least a string?
I'd suggest cleaning up as many as you can.
Or better yet, remove almost all of them and only leave the subset that we are actually interested in
- ID
- Status
- Passed
- Error (would be great to get rid of interface{} here, if possible, otherwise we'll have a hard time handling this one)

Reply:
I checked the doc. `passed` can be 0, 1, null. `error` null or string with error.  I left only 4 field and handle `passed` and `error` properly.